openmdao_book/features/core_features/working_with_components/continuous_variables.ipynb

```diff
@@ -51,6 +51,26 @@
     "        outputs['area'] = inputs['length'] * inputs['width']\n"
```

Comment:
<p>Looks good. Just make this one an assert instead of a print.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks for catching that. Fixed